content/en/blog/detect-threats-falco-ovh-mks-audit-logs-plugin/index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Detecting Threats in OVHcloud MKS Audit Logs with Falco
+date: 2025-03-13
+author: Aurélie Vache
+slug: detect-threats-falco-ovh-mks-audit-logs-plugin
+images:
+  - /blog/detect-threats-falco-ovh-mks-audit-logs-plugin/images/k8saudit-ovh-featured.png
+tags: ["Falco","Kubernetes", "Plugin", "OVHcloud", "MKS", "auditlogs"]
+---
+
+Detecting threats in a Kubernetes cluster can be challenging, we generally don't know where and how to start.
+The good news is that we have an amount of valuable logs that can help us to know what is happened in the cluster. Indeed, each actions, requested or done by a user or an app, in a cluster, are recorded in Audit Logs.
+Kubernetes events are key to understand the behavior of a cluster.
+
+We already provide plugins that let you parse Audit Logs and use Falco to detect threats from GKE, EKS and AKS clusters. With our latest plugin, you'll now have the same powerful threat detection capabilities for your OVHcloud MKS clusters 🎉.
+
+## What is Falco?
+
+Falco is an Open Source cloud-native runtime security tool. It provides near real-time threat detection for cloud, container, and Kubernetes workloads by leveraging runtime insights. Falco can monitor events from various sources, including the Linux kernel, and enrich them with metadata from the Kubernetes API server, container runtime, and more.
+
+Falco can receive **Events**, compare them to a set of **Rules** to determine the actions to perform and generate **Alerts** to different endpoints. 
+
+## What is the OVH MKS Audit Logs plugin?
+
+The OVH audit logs plugin (k8saudit-ovh) extends Falco's capabilities to OVHcloud Managed Kubernetes Service (MKS) clusters, providing you with the same security insights and threat detection Falco already offers for GKE, EKS and AKS environments.
+
+With this plugin, you can seamlessly integrate MKS Audit Logs into Falco's event processing pipeline, enabling it to identify anomalies, suspicious activities, and policy violations within your MKS-based workloads.
+
+Concretely, when an user will execute some **kubectl** commands in an OVHcloud MKS cluster, Audit Logs will be generated. Falco is listening from them and depending on the configured rules to follow, it will generate some alerts.
```

Comment:
```suggestion
With this plugin, you can seamlessly integrate MKS Audit Logs into Falco's event processing pipeline, enabling it to identify anomalies, suspicious activities, and policy violations within your MKS-based workloads.

Concretely, when a user executes some **kubectl** commands in an OVHcloud MKS cluster, Audit Logs will be generated. Falco is listening to them, and depending on the configured rules to follow, it will generate some alerts.
```